packages/manager/modules/iam/src/components/actionSelect/translations/Messages_fr_FR.json

```diff
@@ -3,6 +3,12 @@
   "iam_action_select_action_search_label": "Rechercher une action",
   "iam_action_select_all_actions_description": "Toutes les actions existantes et à venir sur tous les types de produits seront autorisées",
   "iam_action_select_all_actions_label": "Autoriser toutes les actions",
+  "iam_action_select_category_count_selection_many_result_one": "({{ selected }} actions sélectionnées | {{ result }} résultat / {{ total }})",
+  "iam_action_select_category_count_selection_many_result_many": "({{ selected }} actions sélectionnées | {{ result }} résultats / {{ total }})",
+  "iam_action_select_category_count_selection_one_result_one": "({{ selected }} action sélectionnée | {{ result }} résultat / {{ total }})",
+  "iam_action_select_category_count_selection_one_result_many": "({{ selected }} action sélectionnée | {{ result }} résultats / {{ total }})",
+  "iam_action_select_category_count_result_many": "({{ result }} résultats / {{ total }})",
+  "iam_action_select_category_count_result_one": "({{ result }} résultat / {{ total }})",
```

Comment:
You should consider to use` data-ng-pluralize` to simplify plural management

Reply:
Hello, from what I see in documentation of ng-pluralize (https://docs.angularjs.org/api/ng/directive/ngPluralize), it only supports one "count" data. But in this use case, we need two mixed count: we can have both selected items and filtered items. I don't see how to simply this code with ng-pluralize so. And I only extended the previous code that was already using this syntax, which by the way, in that case, would have make sense. But with two count variables, I don't see how to overcome this with ng-pluralize.